tests/test_inference.py

```diff
@@ -4156,41 +4144,39 @@ def test_sample_too_old(self):
 
 class TestAddToSchema:
     def test_is_copy(self):
-        schema = tsinfer.permissive_json_schema()
```

Comment:
A less noisy option would have been to change the definition of ``permissive_json_schema`` to return tskit's version...

Reply:
Yes, but it felt like leaving cruft in the tsinfer codebase, which could have been stripped out.